elixir/test/quizzy_client_test.exs

```diff
@@ -0,0 +1,12 @@
+defmodule QuizzyClientTest do
+  @moduledoc """
+  This is a working ExUnit test. It serves as a
+  placeholder in case you decide to write tests
+  for your solutions.
+  """
+  use ExUnit.Case
+
+  test "Elixir can add numbers" do
```

Comment:
For me it's fine if you implement a very basic relevant unit test here, like counting the events.